snap-css-web/src/App.vue

```diff
@@ -27,4 +39,66 @@
 #nav a.router-link-exact-active {
   color: #42b983;
 }
+
+.shadow-sm {
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.1) !important;
+}
+.shadow {
+  box-shadow: 0 0.35rem 0.75rem rgba(0, 0, 0, 0.07) !important;
+}
+.shadow-lg {
+  box-shadow: 0 1rem 3rem rgba(0, 0, 0, 0.15) !important;
+}
+.shadow-none {
+  box-shadow: none !important;
+}
+.shadow-in {
+  box-shadow: 0 0 5px #dededede inset !important;
```

Comment:
#dededede should be #dedede

Reply:
why tho ??
the last two de apply to the a value in the rgba